src/App.js

```diff
@@ -23,29 +22,45 @@ class App extends React.Component {
     super();
     this.state = {
       list: [
-        // {
-        //   task: 'Organize Garage',
-        //   id: 1528817077286,
-        //   completed: false
-        // },
-        // {
-        //   task: 'Bake Cookies',
-        //   id: 1528817084358,
-        //   completed: false
-        // }
+        {
+          task: 'Organize Garage',
+          id: 1528817077286,
+          completed: true
+        },
+        {
+          task: 'Bake Cookies',
+          id: 1528817084358,
+          completed: false
+        }
       ],
       task: "",
       id: "",
       completed: ""
     };
   }
+  toggleComplete = id => {
+    this.setState(prevState => {
+      return {
+        list: prevState.list.map(task => {
+          if (task.id === id) {
+            return {
+              ...task, completed: !task.completed
+            }
+          } else { return task}
+        })
```

Comment:
Great job breaking down the logic needed to find the task and specifically modify that tasks `completed` status